jmeter/plugins/jmeter-backendlistener-maestro/README.md

```diff
@@ -0,0 +1,63 @@
+Maestro Backend Listener for JMeter
+----
+Maestro Backend Listener is a JMeter plugins that sends sample results to Maestro API.
+
+
+## Sample Result Metrics
+
+- datetime
+- elapsed
+- label
+- datetime
+- elapsed
+- label
+- response_code
+- response_message
+- thread_name
+- data_type
+- success
+- failure_message
+- bytes
+- sent_bytes
+- grp_threads
+- all_threads
+- url
+- latency
+- idle_time
+- connect
+
+
+## Development/Build Environment
+### Prerequisites
+You need to have the following programs installed in your system:
+
+- JDK 1.8
+- Maven 3.8.x
+
+
+### Create JAR package
+
+To create the JMeter BackendListener Maestro plugin JAR package run:
+
+```bash
+mvn package
+```
+
+When the packaging process finishes, the .jar file will be available on the directory `./target`.
+
```

Comment:
Some note aside not related to the changes we have here.

Would be good to have Github action to build the plugin once anything on this folder is changed. 
Maybe we could also publish plugins by using Github packages:
https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven

Or just use artifacts:
https://docs.github.com/en/actions/advanced-guides/storing-workflow-data-as-artifacts

Reply:
Addressed on PR #204